unit_tests/test_utils__operator__multitenant_common.py

```diff
@@ -177,12 +177,12 @@ def _multitenant_class_with_unique_options(self, klass):
         # Process tenant-1 specific options
         for cmd_name in ("stress_cmd_r", "stress_cmd_m"):
             self.assertEqual(
-                tenants[0].params.get(cmd_name),
+                tenants[0].params.get(cmd_name)[0],
```

Comment:
The fact that you need to change this file means that the K8S is broken for sure.